arch/arm/boot/dts/imx6ul-ts7100-custom_05.dtsi

```diff
@@ -0,0 +1,176 @@
+// SPDX-License-Identifier: GPL-2.0+ or X11
+/*
+ * Copyright (C) 2023 Technologic Systems, Inc. dba embeddedTS
+ */
+
+/ {
+	leds {
+		compatible = "gpio-leds";
+
+		led-2 {
+			color = <LED_COLOR_ID_GREEN>;
+			function = LED_FUNCTION_INDICATOR;
+			gpios = <&gpio7 9 GPIO_ACTIVE_LOW>;
+			default-state = "off";
+		};
+
+		led-3 {
+			color = <LED_COLOR_ID_RED>;
+			function = LED_FUNCTION_INDICATOR;
+			gpios = <&gpio7 8 GPIO_ACTIVE_LOW>;
+			default-state = "off";
+		};
+	};
+};
+
+&ecspi3 {
+	num-cs = <1>;
+	cs-gpios = <&gpio4 12 GPIO_ACTIVE_LOW>;
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_ecspi3>;
+	status = "okay";
+
+	wilc3000: wifi@0 {
+		compatible = "microchip,wilc3000";
+		reg = <0>;
+		spi-max-frequency = <20000000>;
+		reset-gpios = <&gpio7 7 GPIO_ACTIVE_HIGH>;
+		chip_en-gpios = <&gpio1 14 GPIO_ACTIVE_HIGH>;
+		interrupt-parent = <&gpio1>;
+		interrupts = <11 GPIO_ACTIVE_HIGH>;
+	};
+};
+
+/*
+ * Differences from TS-7100-Z:
+ *   IO00 (6UL_AN4 on schematic) -> RS485_5_TXEN
+ *   IO09 (6UL_AN3 on schematic) -> RS485_2_TXEN
+ */
```

Comment:
If this is custom, we probably don't want to list all of the differences and such in plaintext.